src/listeners/substrate-listener/commands/genetic-testing/dna-sample-result-ready/dna-sample-result-ready.handler.ts

```diff
@@ -0,0 +1,36 @@
+import { CommandHandler, ICommandHandler } from '@nestjs/cqrs';
+import { DnaSampleResultReadyCommand } from './dna-sample-result-ready.command';
+import { NotificationDto } from '../../../../../endpoints/notification/dto/notification.dto';
+import { NotificationService } from '../../../../../endpoints/notification/notification.service';
+import { DateTimeProxy } from '../../../../../common';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+@CommandHandler(DnaSampleResultReadyCommand)
+export class DnaSampleResultReadyCommandHandler
+  implements ICommandHandler<DnaSampleResultReadyCommand>
+{
+  constructor(
+    private readonly notificationService: NotificationService,
+    private readonly dateTimeProxy: DateTimeProxy,
+  ) {}
+
+  async execute(command: DnaSampleResultReadyCommand) {
+    const dnaSample = command.dnaSample;
+
+    const testResultNotification: NotificationDto = {
+      role: 'Customer',
+      entity_type: 'Genetic Testing Tracking',
+      entity: 'Order Fulfilled',
+      description: `Congrats! Your test results for ${dnaSample.orderId} are out. See your test details.`,
```

Comment:
Your test results for `${dnaSample.orderId}` are out. Click here to see your order details.